sw/airborne/arch/stm32/mcu_periph/can_arch.c

```diff
@@ -78,6 +78,21 @@ void can_hw_init(void)
   nvic_enable_irq(NVIC_USB_LP_CAN_RX0_IRQ);
   nvic_set_priority(NVIC_USB_LP_CAN_RX0_IRQ, NVIC_USB_LP_CAN_RX0_IRQ_PRIO);
 
+#elif STM32F4
+
+/* Enable peripheral clocks. */
+rcc_periph_clock_enable(RCC_GPIOB);
```

Comment:
please indent correctly and would be nicer to use the CAN1_GPIO_PORT_RX etc. defines instead of fixed gpio port/pin as #1185